src/communication.rs

```diff
@@ -19,19 +19,27 @@ pub enum Error {
     MinVersion(&'static str),
 }
 
+#[cfg(not(feature = "enforce-send"))]
 #[async_trait(?Send)]
```

Comment:
why not just remove `(?Send`) here? not sure the new feature is needed. Any downside in making it `Send` for everyone?